cmd/samples/samples_test.go

```diff
@@ -31,9 +31,10 @@ import (
 func TestCreateNativeConfigs(t *testing.T) {
 	print := testutil.Printer("TestCreateNativeConfigs")
 
-	defer os.RemoveAll("./native")
+	tmpDir := "./native"
```

Comment:
You have a number of these hard-coded tmpDirs... should we be using `ioutil.TempDir`?

Reply:
The reason is TempDir creates the directory while we would like the CLI to create the directory. Does a `ioutil.TempDir()` followed by a `os.RemoveAll()` (in addition to the deferred one) sound better to you?

Reply:
Yes, in general I think any directory created during testing should be created in a tempDir.

Reply:
Makes sense. It has been addressed in the latest commit.